WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1016,46 +1012,47 @@ import Combine
             return
         }
 
-        syncContext.perform { [weak self] in
-            guard let topicInContext = (try? self?.syncContext.existingObject(with: topic.objectID)) as? ReaderAbstractTopic else {
-                DDLogError("Error: Could not retrieve an existing topic via its objectID")
-                return
-            }
-
-            let objectID = topicInContext.objectID
+        let objectID = topic.objectID
 
-            let successBlock = { [weak self] (count: Int, hasMore: Bool) in
-                DispatchQueue.main.async {
-                    if let strongSelf = self {
-                        if strongSelf.recentlyBlockedSitePostObjectIDs.count > 0 {
-                            strongSelf.recentlyBlockedSitePostObjectIDs.removeAllObjects()
-                            strongSelf.updateAndPerformFetchRequest()
-                        }
-                        strongSelf.updateLastSyncedForTopic(objectID)
+        let successBlock = { [weak self] (count: Int, hasMore: Bool) in
+            DispatchQueue.main.async {
+                if let strongSelf = self {
```

Comment:
Would this be a good chance to adopt the modern Swift syntax that allows to do 

```swift
if let self = self {
```

?

At the same time, there are 54  usages of this pattern in 21 files, so maybe it's best to batch tackle them.

<img width="406" alt="image" src="https://user-images.githubusercontent.com/1218433/199308353-f0c91e50-9d73-407c-84fe-f24d0cf67fbf.png">

Reply:
I opened https://github.com/wordpress-mobile/WordPress-iOS/issues/19549 to track the wider `strongSelf` discussion.